src/ui/default-prop-types.js

```diff
@@ -173,3 +177,44 @@ export const formItemLayoutShape = PropTypes.shape({
     sm: formItemDimensionShape.isRequired
   }).isRequired
 });
+
+export const importTaskShape = PropTypes.shape({
+  _id: PropTypes.string.isRequired,
+  batchId: PropTypes.string.isRequired,
+  taskType: PropTypes.oneOf([TASK_TYPE.importDocument]),
```

Comment:
We could maybe here say `PropTypes.oneOf(Object.values(TASK_TYPE)` to not need to change it when we add new task types.

Reply:
Ah, true, thanks!